### ISSUE TITLE
Update JDK Versions for CVE-2022-21449

### ISSUE THREAD

**Issue by fredlahde** [adoptium/containers#181]
Do you plan to release updates for the jdk images to include the upstream patches for CVE-2022-21449?

Upstream Advisory: https://openjdk.java.net/groups/vulnerability/advisories/2022-04-19

**Comment by jerboaa** [comment]
Yes, see the banner of the website: https://adoptium.net/ Once builds are done, containers will get updated with them.

**Comment by fredlahde** [comment]
Awesome, thanks! :)

**Comment by vahidshirvani** [comment]
I can see that you have merged the 17.0.3+7 [here](https://github.com/adoptium/containers/commit/c6cafdc0ed1ac976856a3fdd9ead64d60ba26ce3), however the images have not shown up in at docker hub [here](https://hub.docker.com/_/eclipse-temurin?tab=tags&page=1&name=17). Either your pipeline needs to push the images to docker hub or you have to manually do it. Do you have any estimation on when this will be done?

**Comment by fredlahde** [comment]
@vahidshirvani Not that I know of any ETAs, but you can track the progress here:

https://github.com/adoptium/adoptium/issues/140
